en/app.ftl

```diff
@@ -127,6 +127,20 @@ whatsnew-feature-alias-to-mask-snippet = Notice a change? We’re now calling al
 # Thus, if your locale did not make this change, please keep the English version.
 whatsnew-feature-alias-to-mask-description = Notice a change? We’re now calling aliases “masks” to make { -brand-name-firefox-relay } easier to use and open the door for new features.
 
+whatsnew-feature-premium-sweden-heading = { -brand-name-relay-premium } available in Sweden
+# A preview of the full content of `whatsnew-feature-premium-sweden-description`.
+# When translating, please make sure the resulting string is of roughly similar
+# length as the English version.
+whatsnew-feature-premium-sweden-snippet = Upgrade now and get even more protection…
```

Comment:
Could we avoid duplicating content for this? It should be only 4 strings

```
whatsnew-feature-premium-sweden-heading = { -brand-name-relay-premium } available in Sweden
whatsnew-feature-premium-finland-heading = { -brand-name-relay-premium } available in Finland
whatsnew-feature-premium-snippet = Upgrade now and get even more protection…
whatsnew-feature-premium-description = Upgrade now and get even more protection — create unlimited email masks, get a custom email subdomain, and more!
```

It's also likely that we'll need this again in the future.

Reply:
Ah good point: a2b02ea